perceval/backends/git.py

```diff
@@ -634,6 +642,10 @@ def log(self, from_date=None, encoding='utf-8'):
 
         :param from_date: fetch commits newer than a specific
             date (inclusive)
+        :param branches: consider only these branches; when None, consider
```

Comment:
You should remove the section about `gitpath` because this client only works when the given directory is a git repository.